collectors/proc.plugin/sys_class_drm.c

```diff
@@ -652,7 +662,7 @@ static int read_multiline_file(procfile *ff, const char *const pathname, collect
     }
 
     procfile_close(ff);
-    return 2; // error
+    return -3;
```

Comment:
Can we convert these negative numbers to a named macro?

Reply:
I can turn them into a local enum if that helps.

Reply:
But it doesn't really matter, as we only care about returning zero or non-zero value.